automation-tests/test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import fs from 'fs';
+import { step } from 'mocha-steps';
+import path from 'path';
+import {
+  DefaultTreeItem,
+  InputBox,
+  QuickOpenBox
+} from 'wdio-vscode-service';
+import {
+  EnvironmentSettings
+} from '../environmentSettings';
+import {
+  utilities
+} from '../utilities';
+
+describe('Org Creation and Authentication', async () => {
+  const tempProjectName = 'TempProject-OrgCreationAndAuth';
+  // const reuseScratchOrg = false;
+  let projectFolderPath: string = undefined;
+  let prompt: QuickOpenBox | InputBox = undefined;
+  let scratchOrgAliasName: string = undefined;
+
+  step('Set up the testing environment', async () => {
+    // This is "set up the testing environment", not "set up the global variables".
+
+    const tempFolderPath = getTempFolderPath();
+    projectFolderPath = path.join(tempFolderPath, tempProjectName);
+
+    // Clean up the temp folder, just in case there are stale files there.
```

Comment:
This step means that parallel test runs won't be possible, correct?

Reply:
Also, thoughts on using a new folder for each spec vs remove and recreating?

Reply:
Each folder **_is_** specific and named to the test (eg `/automation-tests/e2e-temp/TempProject-DebugApexTests`) (and parallel tests could be run, though sfdx would need to be fixed first to handle crating multiple scratch orgs at the same time)